Project.toml

```diff
@@ -1,18 +1,23 @@
 name = "UnixTimes"
 uuid = "ab1a18e7-b408-4913-896c-624bb82ed7f4"
 authors = ["Christian Rorvik <christian.rorvik@gmail.com>"]
-version = "1.3.0"
+version = "1.3.1"
 
 [deps]
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
-TimeZones = "f269a46b-ccf7-5d73-abea-4c690281aa53"
 
 [compat]
-julia = "1"
-TimeZones = "1"
```

Comment:
Compat entries are still relevant for weak dependencies, otherwise we could get a situation where TimeZones is loaded but the extension fails to work due to breaking API changes

Reply:
You're right, I've corrected this.